src/base1/patternfly-4-overrides.scss

```diff
@@ -0,0 +1,39 @@
+/*** PF4 overrides ***/
+
+/* WORKAROUND: Override word-break bug */
+/* See: https://github.com/patternfly/patternfly-next/issues/2325 */
+.pf-c-table td {
+    word-break: normal;
+    overflow-wrap: break-word;
+}
+
+/* WORKAROUND:  Dropdown (PF4): Caret is not properly aligned bug */
+/* See: https://github.com/patternfly/patternfly/issues/2715 */
+/* Align the icons inside of all dropdown toggles. */
+/* Part 1 of 2 */
+.pf-c-dropdown__toggle-button {
+  display: flex;
+  align-items: center;
+}
+
+/* Make split button dropdowns the same height as their sibling. */
+/* Part 2 of 2 */
+.pf-m-split-button {
+  align-items: stretch;
+}
```

Comment:
I just checked. It's pulled in, but perhaps the order is incorrect? I see it referenced, but with a strikeout... `.pf-c-dropdown__toggle`, which comes from PatternFly, overrides it.

`.pf-m-split-button` from our overrides file should come _after_ PatternFly CSS, so it can override it.

(They're both the same specificity level, so whatever comes later wins in this case. That's how I can tell that the overrides must currently come before, which is incorrect.)